### PR TITLE
added slider component to my component library

### PR DESCRIPTION
**Pull request by Hamza-777**
preview: https://zen-chandrasekhar-c19bab.netlify.app/docs.html#sliders

### CODE REVIEW

File: components/sliders.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.slidecontainer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (ShekharDhangar) [inline components/sliders.css:5]: use the properties as helper class and include all 3 main proerties in one class and then use it

**Review by ShekharDhangar — APPROVED** [review]
Do the suggested changes, rest looks good

File: components/sliders.css (hunk 1 above)
Comment (Hamza-777) [reply components/sliders.css:5]: Okay! Will do it.

**Review by md-amir-gauhar — APPROVED** [review]

File: components/sliders.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    border: 2px solid var(--primary-color);
+    border-radius: 50%;
+    background: var(--white-color);
+    cursor: pointer;
+}
+
+.slider::-moz-range-thumb {
+    width: 25px;
+    height: 25px;
+    background: var(--primary-color);
```
Comment (md-amir-gauhar) [inline components/sliders.css:44]: Try to stay consistent with css units

**Review by md-amir-gauhar — APPROVED** [review]

File: components/sliders.css (hunk 2 above)
Comment (Hamza-777) [reply components/sliders.css:44]: will do.

**Review by HrishikeshMahalle — APPROVED** [review]
The code looks good to merge. Although try to stay consistent with the units.

**Pull request merged**